### ISSUE TITLE
clearml - WARNING - Error: I/O operation on closed file ，PIL Image（lazy load),but fd closed before read in Tensorflow_bind

### ISSUE THREAD

**Issue by AncientRemember** [allegroai/clearml#914]
https://github.com/allegroai/clearml/blob/311e05d9384dd470a7a1918fde0c6e770b7f15a8/clearml/binding/frameworks/tensorflow_bind.py#L401

```
try:
    if isinstance(img_str, bytes):
        imdata = img_str
    else:
        imdata = base64.b64decode(img_str)
    output = BytesIO(imdata)
    im = Image.open(output)
    # if this is a GIF store as is
    if getattr(im, 'is_animated', None):
        output.close()
        fd, temp_file = mkstemp(
            suffix=guess_extension(im.get_format_mimetype()) if hasattr(im, 'get_format_mimetype')
            else ".{}".format(str(im.format).lower())
        )
        with open(fd, "wb") as f:
            f.write(imdata)
        return temp_file
    output.close()
    if height is not None and height > 0 and width is not None and width > 0:
        val = np.array(im).reshape((height, width, -1)).astype(np.uint8)
    else:
        val = np.array(im).astype(np.uint8)
    if val.ndim == 3 and val.shape[2] == 3:
        if self._visualization_mode == 'BGR':
            val = val[:, :, [2, 1, 0]]
        else:
            val = val
    elif (val.ndim == 2) or (val.ndim == 3 and val.shape[2] == 1):
        val = np.tile(np.atleast_3d(val), (1, 1, 3))
    elif val.ndim == 3 and val.shape[2] == 4:
        if self._visualization_mode == 'BGR':
            val = val[:, :, [2, 1, 0]]
        else:
            val = val[:, :, [0, 1, 2]]
    
```

**Comment by alex-burlacu-clear-ml** [comment]
Hey @AncientRemember ! Could you please add the following details to this issue, to help us fix it?

- What version of OS, Python, clearml, tensorflow, and tensorboard were used when this warning was emitted?
- What kind of images are you trying to report with tensorboard (image file format and size specifically)?
- A snippet of the image reporting code which causes this warning message

Also, just in case, check out our [issue reporting checklist](https://github.com/allegroai/clearml/blob/master/docs/contributing.md#reporting-issues)

**Comment by alex-burlacu-clear-ml** [comment]
This issue was fixed in the latest (1.9.2) release of the clearml SDK. I'm marking this issue as closed, but if you find that the issue persists, please feel free to reopen it.
